css/index.css

```diff
@@ -113,3 +316,13 @@ body * {
 .self-center {
   align-self: center;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle, a few of our CSS properties have been repeated, for example; `flex-direction: column`, `text-align: center`

Reply:
Hello @addod19,

Thank you for the awesome suggestion, t has helped me take a deep understanding of the CSS DRY principle.